standard_icone_U13631771_202207122127_1657663329.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Should these "standard_icone" example files be under a data directory? Or do we want to exclude/remove altogether if not used?

Reply:
good catch, moved these to the sample_files directory.